### DISCUSSION TITLE
Support winget package manager

### DISCUSSION THREAD

**Discussion by nascosto** [Versent/saml2aws#1425] [Ideas and Feature Requests]
[winget ](https://github.com/microsoft/winget-cli)is now a common package management tool for Windows. It would be great to have winget support for this package. This can be done as part of the pipeline: https://github.com/microsoft/winget-create/blob/main/README.md#using-windows-package-manager-manifest-creator-in-a-cicd-pipeline

**Comment by krisek** [comment]
https://github.com/microsoft/winget-pkgs/pull/246355
